packages/server/src/lib/gost-aws.js

```diff
@@ -0,0 +1,72 @@
+const AWS = require('aws-sdk');
+
+/*
+----------------------------------------------------------
+                        AWS S3
+----------------------------------------------------------
+*/
+
+function getS3Client() {
+    let s3;
+    if (process.env.LOCALSTACK_HOSTNAME) {
+        /*
+            1. Make sure the local environment has awslocal installed.
+            2. Use the commands to create a bucket to test with.
+                - awslocal s3api create-bucket --bucket arpa-audit-reports --region us-west-2 --create-bucket-configuration '{"LocationConstraint": "us-west-2"}'
+            3. Access bucket resource metadata through the following URL.
+                - awslocal s3api list-buckets
+                - awslocal s3api list-objects --bucket arpa-audit-reports
+        */
+        console.log('------------ USING LOCALSTACK ------------');
+        const endpoint = new AWS.Endpoint(`http://${process.env.LOCALSTACK_HOSTNAME}:${process.env.EDGE_PORT || 4566}`);
+        s3 = new AWS.S3({
+            endpoint,
+            s3ForcePathStyle: true,
+        });
+    } else {
+        s3 = new AWS.S3();
+    }
+    return s3;
+}
+
+/*
+----------------------------------------------------------
+                        AWS SES
+----------------------------------------------------------
+*/
+
+function sendEmail(message) {
+    if (process.env.SUPPRESS_EMAIL) return;
+
+    const transport = new AWS.SES();
+    const params = {
+        Destination: {
+            ToAddresses: [message.toAddress],
+        },
+        Source: process.env.NOTIFICATIONS_EMAIL,
+        Message: {
+            Subject: {
+                Charset: 'UTF-8',
+                Data: message.subject,
+            },
+            Body: {
+                Html: {
+                    Charset: 'UTF-8',
+                    Data: message.body,
+                },
+                Text: {
+                    Charset: 'UTF-8',
+                    Data: message.text,
+                },
+            },
+        },
+    };
+    transport.sendEmail(params).promise()
+        .then((data) => console.log('Success sending SES email:', data))
+        .catch((err) => console.error('Error sending SES email:', err, err.stack));
+}
```

Comment:
No change needed for this right now, but noting for future reference: I'm thinking it would be better for the `.catch()` path to log and re-throw the error so that callers of `sendEmail()` are responsible for handling the error in context of the operation that's sending a specific email (which may include re-trying, formulating a response based on the error, etc.).

Reply:
That would also make this more testable - we can directly test for the contents of an error, but can't check a console log without returning the promise from this function.